prod-cockpit-rcar.yaml

```diff
@@ -614,3 +614,15 @@ parameters:
             builder:
               env:
                 - "ANDROID_EVS_ENABLED=true"
+  ENABLE_SCMI:
```

Comment:
Hi @iusyk, please remove this parameter. It is better not to add the parameter that doesn't work.

Reply:
reason ?

Reply:
Done.